rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java

```diff
@@ -147,6 +149,13 @@
     public InfrastructureManager() {
     }
 
+    @Override
+    public Map<String, String> getMeta() {
+        Map<String, String> meta = new HashMap<>();
```

Comment:
Souldn't that hashmap be configured once at object allocation instead of recreating it each time getMeta() is invoked ?